flipperdevices/flipperzero-firmware
# [FL-1235] Cut long names

# What's new

- NEW: `elements_string_fit_width` - fit string to desired width in pixels
- added `elements_string_fit_width ` to `file select` module  and `archive` app
- fix typo across Archive app

# Verification 

- Create few keys with loooong names
- Check archive/saved menu in ibutton app to verify

# Checklist (do not modify)
